plugin/src/main/kotlin/tanvd/grazi/EnglishChonker.kt

```diff
@@ -0,0 +1,44 @@
+package tanvd.grazi
+
+import com.intellij.openapi.application.ApplicationManager
+import org.jetbrains.org.objectweb.asm.*
+import org.jetbrains.org.objectweb.asm.Opcodes.ASM5
+
+
+class EnglishChonker(cv: ClassVisitor) : ClassVisitor(ASM5, cv) {
```

Comment:
Let's rename it to `EnglishChunkerInstrumentation` and move into LangToolInstrumentation object

Reply:
Done